unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/AssetManager/AssetBundles/AB/Providers/EditorAssetBundleProvider.cs

```diff
@@ -0,0 +1,24 @@
+using Cysharp.Threading.Tasks;
+using System.IO;
+using System.Threading;
+using UnityEngine;
+
+namespace DCL.Providers
+{
+    /// <summary>
+    /// Local Asset Bundles support, if you have the asset bundles in that folder, we are going to load them from there.
+    /// This is a debug feature
+    /// </summary>
+    public class EditorAssetBundleProvider : IAssetBundleProvider
```

Comment:
I was not aware of this functionality, let's reevaluate if it's used or needed and remove it if not.

Reply:
To be honest, we usually create a local content server and place the ABs there, this is easier to do. Let's keep it and reevaluate in the future.

Reply:
To me, it was helpful with the CJK fonts. Before uploading the assets to web, I used this functionality to check if the fonts were being imported correctly.